core/amber/src/main/scala/edu/uci/ics/texera/web/resource/auth/GoogleAuthResource.scala

```diff
@@ -32,6 +32,12 @@ object GoogleAuthResource {
 
 @Path("/auth/google")
 class GoogleAuthResource {
+  final private lazy val clientId = AmberUtils.amberConfig.getString("user-sys.googleClientId")
+  @GET
+  @Path("/clientid")
+  def getClientId: String = {
+    clientId
+  }
```

Comment:
Could you please refresh my mind that if storing and distributing client id from server a good idea? Maybe share some documents that suggest this way?

Note that this Resource is not protected by role system so anyone can get it.

Reply:
Client ID is publicly available.
Reference:
https://www.googlecloudcommunity.com/gc/Apigee/Client-Id-vs-Client-Secret/m-p/13501#:~:text=Client%20ID%20is%20publicly%20available,securely%20%26%20not%20available%20to%20public.